notification/src/utils/hmacValidator.js

```diff
@@ -18,10 +18,19 @@ function hasValidHmacSignature (notification) {
   }
   */
   const notificationRequestItem = notification.NotificationRequestItem
-  if (!notificationRequestItem.additionalData || !notificationRequestItem.additionalData.hmacSignature)
-    return false
-
-  return validator.validateHMAC(notificationRequestItem, config.adyen.secretHmacKey)
+  if (!notificationRequestItem.additionalData)
+    return 'Notification does not contain the required field '
```

Comment:
I am not sure if this field is required.
https://docs.adyen.com/development-resources/webhooks/notifications-api#notificationrequestitem

Reply:
I also could not see a test case for that?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/465/commits/2dcac27a0eb89adfdc404c0d99be0ed1959c7195

Reply:
It's required if you enable HMAC. Check where it is being called: https://github.com/commercetools/commercetools-adyen-integration/pull/465/files#diff-03c88d579df3d6ccea32a895f34ddf21ac83c4b6a0d8809be06d1902df7e76c3R18